internal/cmd/db_generatetoken.go

```diff
@@ -12,7 +13,7 @@ var readOnly bool
 func init() {
 	dbTokensCmd.AddCommand(dbGenerateTokenCmd)
 
-	usage := "Token expiration. Possible values are 'default' or 'none'."
+	usage := fmt.Sprintf("Token expiration. Possible values are %s (default) or expiration time in days (e.g. %s).", internal.Emph("never"), internal.Emph("7d"))
```

Comment:
I would stick to none instead of never. Even though never is a more clear.

Reply:
I would allow more time units not just days.

Reply:
As said in the API's PR, kept it with days for now and changed back `never` to `none`.